src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleaseController.cs

```diff
@@ -229,6 +254,26 @@ private async Task<ActionResult> CheckReleaseExistsAsync(ReleaseId releaseId, Fu
 
             return Ok(result.Right);
         }
+        
+        private async Task<ActionResult> CheckReleaseExistsStreamAsync(ReleaseId releaseId,
+            Func<Task<Either<ValidationResult, FileStreamResult>>> andThen)
+        {
+            var release = await _releaseService.GetAsync(releaseId);
+            if (release == null)
+            {
+                return NotFound();
+            }
+
+            var result = await andThen.Invoke();
+            if (result.IsLeft)
+            {
+                ValidationUtils.AddErrors(ModelState, result.Left);
+                return ValidationProblem();
```

Comment:
I think we just want to return a 404 here, given that its a image endpoint the validation is little use.

Reply:
So the reason I've not done this is that currently we just get a validation result back from the service layer, Now we could do some mapping of validation messages to response codes and I reckon it'd definitely be worth doing at some point -  but it'll be work....